connectors/sources/sharepoint_server.py

```diff
@@ -35,13 +39,14 @@
 SELECTED_FIELDS = "WikiField, Modified,Id,GUID,File,Folder"
 
 URLS = {
-    PING: "{host_url}{parent_site_url}_api/web/webs",
-    SITES: "{host_url}{parent_site_url}/_api/web/webs?$skip={skip}&$top={top}",
-    LISTS: "{host_url}{parent_site_url}/_api/web/lists?$skip={skip}&$top={top}&$expand=RootFolder&$filter=(Hidden eq false)",
-    ATTACHMENT: "{host_url}{value}/_api/web/GetFileByServerRelativeUrl('{file_relative_url}')/$value",
-    DRIVE_ITEM: "{host_url}{parent_site_url}/_api/web/lists(guid'{list_id}')/items?$select={selected_field}&$expand=File,Folder&$top={top}",
-    LIST_ITEM: "{host_url}{parent_site_url}/_api/web/lists(guid'{list_id}')/items?$expand=AttachmentFiles&$select=*,FileRef",
-    ATTACHMENT_DATA: "{host_url}{parent_site_url}/_api/web/getfilebyserverrelativeurl('{file_relative_url}')",
+    PING: "{site_url}/_api/web/webs",
+    SITE: "{parent_site_url}/_api/web",
```

Comment:
Out of curiosity - why did SITE url changed from `_api/web/webs` to `_api/web`?

Reply:
It didn't change, it's an addition similar to SITES. I need to get the whole SPWeb object for the current site and all it's subsites in order to access the relative and full URLs of the sites.

Reply:
Ah okay, can you send an example jsob object that it returns if possible for our functional tests?

Reply:
So a full response looks like this:
`{
   "d":{
      "AllowRssFeeds":true,
      "AlternateCssUrl":"",
      "AppInstanceId":"00000000-0000-0000-0000-000000000000",
      "Configuration":0,
      "Created":"2023-10-24T00:24:36",
      "CurrentChangeToken":{
         "__metadata":{
            "type":"SP.ChangeToken"
         },
         "StringValue":"1;2;cc594a12-3ded-451a-b868-40ccec07d354;638495628235200000;40440"
      },
      "CustomMasterUrl":"/_catalogs/masterpage/seattle.master",
      "Description":"Portal Home",
      "DesignPackageId":"00000000-0000-0000-0000-000000000000",
      "DocumentLibraryCalloutOfficeWebAppPreviewersDisabled":false,
      "EnableMinimalDownload":true,
      "HeaderEmphasis":0,
      "HorizontalQuickLaunch":false,
      "Id":"cc594a12-3ded-451a-b868-40ccec07d354",
      "IsMultilingual":false,
      "Language":1033,
      "LastItemModifiedDate":"2024-04-24T13:40:24Z",
      "LastItemUserModifiedDate":"2024-04-24T11:33:33Z",
      "MasterUrl":"/_catalogs/masterpage/seattle.master",
      "NoCrawl":false,
      "OverwriteTranslationsOnChange":false,
      "ResourcePath":{
         "__metadata":{
            "type":"SP.ResourcePath"
         },
         "DecodedUrl":"https://nsp1"
      },
      "QuickLaunchEnabled":true,
      "RecycleBinEnabled":true,
      "ServerRelativeUrl":"/",
      "SiteLogoUrl":null,
      "SyndicationEnabled":true,
      "Title":"Portal Home",
      "TreeViewEnabled":false,
      "UIVersion":15,
      "UIVersionConfigurationEnabled":false,
      "Url":"https://nsp1",
      "WebTemplate":"STS",
      "WelcomePage":"SitePages/Home.aspx"
   }
}`

But you only care for a few of these, plus some other values that will be useful in the future:

`{
   "d":{
      "Created":"2023-10-24T00:24:36",
      "Description":"Portal Home",
      "Id":"cc594a12-3ded-451a-b868-40ccec07d354",
      "Language":1033,
      "LastItemModifiedDate":"2024-04-24T13:40:24Z",
      "NoCrawl":false,
      "ServerRelativeUrl":"/",
      "Title":"Portal Home",
      "Url":"https://nsp1",
      "WebTemplate":"STS",
      "WelcomePage":"SitePages/Home.aspx",
	  "HasUniqueRoleAssignments":false
   }
}`